.gitignore

```diff
@@ -1 +1,22 @@
-/target
+target/
+*.class
+*.log
+metrics/
+#mvn build
+#vert.x
+.vertx/
+file-uploads/
+#netbeans
+nbproject/
+#eclipse
+.project
+.settings/
+.classpath
+#idea
+.idea/
+*.iml
+#npm
+node_modules
+.factorypath
+.vscode/
+out
```

Comment:
Wow, that is a hell of a lot of stuff to ignore!

Reply:
yes